src/main/java/frc/robot/subsystems/swervedrive/TunerConstants.java

```diff
@@ -90,36 +90,36 @@ public class TunerConstants {
     private static final int kFrontLeftSteerMotorId = 6;
     private static final int kFrontLeftEncoderId = 13;
     private static final double kFrontLeftEncoderOffset = 0.211181640625;
-
-    private static final double kFrontLeftXPosInches = 6.5;
-    private static final double kFrontLeftYPosInches = 6.5;
+    private static final double kSwerveMotorOffset = 2.625;
+    private static final double kFrontLeftXPosInches = 14 - kSwerveMotorOffset;
+    private static final double kFrontLeftYPosInches = 14 - kSwerveMotorOffset;
 
     // Front Right
     private static final int kFrontRightDriveMotorId = 7;
     private static final int kFrontRightSteerMotorId = 8;
     private static final int kFrontRightEncoderId = 12;
     private static final double kFrontRightEncoderOffset = -0.015380859375;
 
-    private static final double kFrontRightXPosInches = 6.5;
-    private static final double kFrontRightYPosInches = -6.5;
+    private static final double kFrontRightXPosInches = 14 - kSwerveMotorOffset;
+    private static final double kFrontRightYPosInches = -14 - kSwerveMotorOffset;
```

Comment:
Confirm sign. -14 - 2.625 = -16.625 which is outside the frame; same with all the negatives.